immuno_tabular/code/make_clean_data.R

```diff
@@ -141,31 +141,37 @@ ds3 <- bind_cols(
     do.call(cbind, .)
 )
 
-grp_lev <- c("", "Age < 65",  "Age >= 65", "At-risk", "Not at-risk", 
-             "Age < 65 At-risk", "Age < 65 Not at-risk",
-             "Age >= 65 At-risk", "Age >= 65 Not at-risk", "Female", "Male", 
-             "Age < 65 Female", "Age < 65 Male", 
-             "Age >= 65 Female", "Age >= 65 Male", 
+grp_lev <- c("", "Age $<$ 65",  "Age $\\geq$ 65", "At-risk", "Not at-risk", 
+             "Age $<$ 65 At-risk", "Age $<$ 65 Not at-risk",
+             "Age $\\geq$ 65 At-risk", "Age $\\geq$ 65 Not at-risk", "Male", "Female", 
+             "Age $<$ 65 Female", "Age $<$ 65 Male", 
+             "Age $\\geq$ 65 Male", "Age $\\geq$ 65 Female", 
              "Hispanic or Latino", "Not Hispanic or Latino", "Not reported and unknown ", 
-             "White Non-Hispanic", "Black or African American", "Asian", 
+             "White Non-Hispanic ", "Black or African American", "Asian", 
              "American Indian or Alaska Native", 
              "Native Hawaiian or Other Pacific Islander", 
              "Multiracial", "Other", "Not reported and unknown",  
-             "Communities of Color", 
-             "Age < 65 White Non-Hispanic", "Age < 65 Communities of Color", 
-             "Age >= 65 Communities of Color", "Age >= 65 White Non-Hispanic")
+             "Communities of Color", "White Non-Hispanic",
+             "Age $<$ 65 Communities of Color", "Age $<$ 65 White Non-Hispanic",  
+             "Age $\\geq$ 65 Communities of Color", "Age $\\geq$ 65 White Non-Hispanic")
 
 ds <- mutate(ds3, Group = factor(subgroup_cat, levels = grp_lev))
 
-resp_v <- grep("Resp|FR2|FR4|2llod|4llod", names(ds), value = T)
+resp_v <- setdiff(grep("Resp|FR2|FR4|2llod|4llod", names(ds), value = T),
+                  grep("liveneut", names(ds), value = T)
+                  )
 
 ds_resp_l <- pivot_longer(ds,
                           cols = all_of(resp_v), 
                           names_to = "resp_cat",
                           values_to = "response") %>%
   inner_join(labels_all, by = "resp_cat")
 
-mag_v <- c(bAb_v, pnAb_v, lnAb_v, grep("DeltaDay", names(ds), value = T))
+mag_v <- setdiff(
```

Comment:
@cyu-hvtn How are you determining which assays to make tables for?

In general, we would like the flexibility to read in the [`assay` variable from `_common.R`](https://github.com/CoVPN/correlates_reporting/blob/d21923079fac6130600f15ffdd04101da9dadbbc/_common.R#L18). This will be helpful for making interim reports as we await full results from the labs.

Reply:
I used to pull it from `assay` from _common.R. Then I noticed that it didn't include the "Anti N IgG" marker, which should be included in the immuno tables but not the figures per Peter's email on 01/21. So I hard coded the assays. I might have missed new discussion on what markers to include?

Reply:
Is it possible for you to instead read in `assay` and simply `c` the Anti N IgG marker. Thus, one could change `assay` definition in `_common.R`, but still have Anti N IgG marker read by your code?

Reply:
Yes, of course!

Reply:
Actually, it would be better to define a variable in `_common.R` that flags whether Anti N IgG is present. So adding something to `_common.R` like:

```r
assays <- ...
include_NIgG <- TRUE
```

And then your code could determine the assays to make tables for based on `assays` (which would not include N IgG) and the `include_NIgG` flag.